src/main/java/br/com/mercadolivre/loja/user/UserForm.java

```diff
@@ -24,7 +28,8 @@ public String getPassword() {
     }
 
     public void setPassword(String password) {
-        this.password = password;
+        BCryptPasswordEncoder passwordEncoder = new BCryptPasswordEncoder();
+        this.password = passwordEncoder.encode(password);
     }
 
     public User toModel() {
```

Comment:
Aqui pode ter o modificador acesso default.

```suggestion
    User toModel() {
```